src/test/signal/test_signals.rs

```diff
@@ -888,6 +889,61 @@ fn test_sigaltstack_autodisarm() -> Result<(), Box<dyn Error>> {
     Ok(())
 }
 
+static GLOBAL_STATIC: u32 = 0xdeadbeef;
+extern "C" fn change_rax_from_null_to_global_static(
+    signal: i32,
+    info: *mut libc::siginfo_t,
+    voidctx: *mut std::ffi::c_void,
+) {
+    assert_eq!(signal, signal::SIGSEGV as i32);
+
+    let info = unsafe { info.as_mut().unwrap() };
```

Comment:
I don't think this reference needs to be mutable?

Reply:
Done